fvm/utils/view.go

```diff
@@ -114,6 +115,7 @@ type MapLedger struct {
 	Registers       map[flow.RegisterID]flow.RegisterValue
 	RegisterTouches map[flow.RegisterID]struct{}
 	RegisterUpdated map[flow.RegisterID]struct{}
+	mu              sync.RWMutex
```

Comment:
nit: if you have a non-pointer mutex you generally want to protect a structure from being copied on a compiler level.

Reply:
(alternatively you can just embed the mutex (since it protects the whole structure) so that all linters would catch an attempt to copy a structure with a `Lock()` method.)

Reply:
64c4c71